modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryListenedCategoriesView.kt

```diff
@@ -1,36 +1,141 @@
-            modifier = modifier
-                .fillMaxWidth()
-                .padding(16.dp)
-        )
+    val backgroundColor = if (story.listenedCategories.isEmpty()) {
+        Podcast().getTintColor(false)
+    } else {
+        // Get the middle podcast
+        story.listenedCategories.take(2).last().toPodcast()
+            .getTintColor(false)
     }
+
+    Column(
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = Color(backgroundColor))
+            .verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = modifier.height(40.dp))
+
+        Spacer(modifier = modifier.weight(0.7f))
+
+        PodcastCoverStack(story)
+
+        Spacer(modifier = modifier.weight(0.3f))
+
+        PrimaryText(story)
+
+        Spacer(modifier = modifier.weight(0.25f))
+
+        SecondaryText(story)
+
+        Spacer(modifier = modifier.weight(0.8f))
+
+        Logo()
+
+        Spacer(modifier = modifier.height(40.dp))
+    }
+}
+
+@Composable
+private fun PodcastCoverStack(
+    story: StoryListenedCategories,
+    modifier: Modifier = Modifier,
+) {
+    val context = LocalContext.current
+    val currentLocalView = LocalView.current
+    val coverWidth = (currentLocalView.width.pxToDp(context).dp) / 2.5f
+    val translateBy = (coverWidth.value * .2).toInt().dpToPx(context)
+
+    Box {
+        for (index in (0..story.listenedCategories.lastIndex).reversed()) {
+            Box(
+                modifier = modifier
+                    .padding(top = (index * (coverWidth.value * .25)).dp)
+                    .transformPodcastCover()
+                    .graphicsLayer(translationX = -translateBy.toFloat())
+            ) {
+                PodcastImage(
+                    uuid = story.listenedCategories[index].mostListenedPodcastId,
+                    dropShadow = false,
+                    modifier = modifier.size(coverWidth)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun PrimaryText(
+    story: StoryListenedCategories,
+    modifier: Modifier = Modifier,
+) {
+    TextH20(
+        text = stringResource(
+            id = LR.string.end_of_year_story_listened_to_categories,
+            story.listenedCategories.count()
+        ),
+        textAlign = TextAlign.Center,
+        color = story.tintColor,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 40.dp)
+    )
+}
+
+@Composable
+private fun SecondaryText(
+    story: StoryListenedCategories,
+    modifier: Modifier = Modifier,
+) {
+    TextP40(
+        text = stringResource(id = LR.string.end_of_year_story_listened_to_categories_subtitle),
+        textAlign = TextAlign.Center,
+        color = story.tintColor,
+        fontWeight = FontWeight.Bold,
+        modifier = modifier
+            .fillMaxWidth()
+            .alpha(0.8f)
```

Comment:
With some tint colors, this `alpha` can make the text a bit hard to read (to my eyes at least).

<img width="350px" src="https://user-images.githubusercontent.com/4656348/201121729-627df21b-c6a3-4ea9-acf6-9e8e622267cf.png" />

Reply:
I'll try adding a background gradient to improve this in a future PR. 

I also realized we need to display only three podcast covers. This is fixed in 82c7c52.